PhysicsTools/NanoAOD/plugins/EventStringOutputBranches.h

```diff
@@ -0,0 +1,40 @@
+#ifndef PhysicsTools_NanoAOD_EventStringOutputBranches_h
```

Comment:
@peruzzim is there a special reason not to have this file integrated inside the ```.cc``` plugin implementation?

Reply:
I did it because I am including this header in `PhysicsTools/NanoAOD/plugins/NanoAODOutputModule.cc`.
If you have a suggestion on how this should be done differently, please let me know and I'll implement it in the next PR.

Reply:
@peruzzim ok, you are correct, I agree